checklist.md

```diff
@@ -45,7 +45,7 @@
 >**WRONG WAY**
 >```scss
 >&__buy-link {
->  display: flex;
+>  display: grid;
 >  margin-top: 20px;
```

Comment:
Why do you change this file?

Reply:
![image](https://user-images.githubusercontent.com/120336257/214412946-c63e5910-403f-4632-ba96-05af4f77d215.png)

Reply:
![image](https://user-images.githubusercontent.com/120336257/214413210-6d1eefec-4b39-4a5b-bc24-5052fabdda0a.png)

Reply:
![image](https://user-images.githubusercontent.com/120336257/214413845-6ccd51c6-9ab5-418f-88d6-98d93c312156.png)

Reply:
![image](https://user-images.githubusercontent.com/120336257/214414098-f71662dc-c005-4c66-bce7-df81c2212ab0.png)